### ISSUE TITLE
Remove postgres and redis port mapping from docker-compose.yml

### ISSUE THREAD

**Issue by Flightkick** [JaneJeon/blink#805]
The `docker-compose.yml` example exposes the ports for redis and postgres publicly.
Declaring the port mapping is not necessary when containers reside in the same network so they can safely be removed.

**Comment by JaneJeon** [comment]
Just for confirmation, you're talking about the one in `deploy/` folder and not the top-level compose.yaml (which is for development) right?

In that case, yeah, good call. Someone else made it, I accepted it without too much scrutiny but it 100% tracks. Mind making a quick PR?

**Comment by Flightkick** [comment]
That is correct! FYI You could also lock down the development one to bind to the loopback adapter instead of the public interface by specifying it in the mapping, e.g.: `- "127.0.0.1:5432:5432"`

I'm not using the project yet and rather not go through the fork/PR process for such a small one-off change. Feel free to pick it up whenever you're touching the code anyway. :)
